src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -331,7 +385,60 @@
   end
 
   describe '#setup_codemirror_editor' do
-    skip('Please add some tests')
+    it 'sets the style for the editor' do
+      allow_any_instance_of(Webui::WebuiHelper).to receive(:content_for).with(:content_for_head, kind_of(String))
+      expect_any_instance_of(Webui::WebuiHelper).to receive(:content_for).with(:head_style, kind_of(String))
+      setup_codemirror_editor
+    end
+
+    it 'increases the uid' do
+      allow_any_instance_of(Webui::WebuiHelper).to receive(:content_for)
+      expect(setup_codemirror_editor).to eq(0)
+      expect(setup_codemirror_editor).to eq(1)
+    end
+
+    it 'sets the javascript include tag' do
+      allow_any_instance_of(Webui::WebuiHelper).to receive(:content_for).with(:head_style, kind_of(String))
+      expect_any_instance_of(Webui::WebuiHelper).to receive(:content_for).with(:content_for_head, kind_of(String))
+
+      setup_codemirror_editor
+    end
+  end
+
+  describe '#codemirror_style' do
+    context 'option height' do
+      it 'uses auto as default value' do
+        expect(codemirror_style).to include('height: auto;')
+      end
+
+      it 'sets it properly' do
+        expect(codemirror_style(height: '250px')).to include('height: 250px;')
+      end
+    end
+
+    context 'option width' do
+      it 'uses auto as default value' do
+        expect(codemirror_style).to include('width: auto;')
+      end
+
+      it 'sets it properly' do
```

Comment:
I would rather use `get set properly` because then you can read it as a sentence: `#codemirror_style option height get set properly`